src/test/java/eu/europa/ted/efx/EfxTemplateTranslatorTest.java

```diff
@@ -157,15 +157,15 @@ void testShorthandFieldValueLabelReferenceForText() {
   @Test
   void testShorthandContextLabelReference_WithValueLabelTypeAndIndicatorField() {
     assertEquals(
-        "declare block01 = { label(concat('indicator', '|', 'value', '-', ./normalize-space(text()), '|', 'BT-00-Indicator')) }\nfor-each(/*/PathNode/IndicatorField).call(block01)",
-        translate("{BT-00-Indicator}  #{value}"));
+        "declare block01 = { label(concat('field', '|', 'name', '|', 'BT-00-Indicator')) }\nfor-each(/*/PathNode/IndicatorField).call(block01)",
```

Comment:
Not sure about what is done here, why is it using field name instead of indicator?
Is this because it has to do with an older version?

Reply:
The semantics changed. Value had spacial semantics in $value and #value
So I wanted to maintain the same semantics in #{value}.
But #{name} is a different story... So I had to fix the expected value tests.